index.html

```diff
@@ -11,10 +11,12 @@
 </head>
 <body>
 <header>
-  <nav class="header2">
-    <div class="header2__logo"><a href="#"><img src="./images/logo.png" alt="logo"></a>
-    </div>
-    <div class="header2__menu"><a href="#"><img src="./icons/menu.png" alt="menu"></a>
+  <div class="logo"><a href="#"><img src="./images/logo.png" alt="logo"></a></div>
+  <nav>
+    <div class="menu">
+      <a href="#"><span>Portfolio</span></a>
+      <a href="#"><span>About</span></a>
+      <a href="#"><span>Contact</span></a>
     </div>
   </nav>
```

Comment:
Figma Design
![Screenshot from 2022-04-30 21-32-35](https://user-images.githubusercontent.com/77842237/166118208-2488fa0e-7ebd-41d6-8ac6-f4a0da5a2d7a.png)
Your Design
![Screenshot from 2022-04-30 21-28-56](https://user-images.githubusercontent.com/77842237/166118233-987295a6-62f3-4f95-8d1b-6864cee86f55.png)


- [ ] Kindly delete the line that underlines the link in the navbar